### ISSUE TITLE
Support "Ctrl+Shift+Z" to Redo

### ISSUE THREAD

**Issue by moonmehta** [ONLYOFFICE/DesktopEditors#180]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Doesn't support universal keyboard shortcut for Redo i.e. `Ctrl+Shift+Z`

**What is the expected behavior?**
`Ctrl+Shift+Z` should redo, `Ctrl+Y` isn't as intuitive.

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
v 5.2.8

**Comment by Rita-Bubnova** [comment]
Hello, @uncertainquark.
We try to support keyboard shortcuts used in MS Office.
The operation Redo is performed using the `Ctrl+Y` key combination.

**Comment by ShockwaveNN** [comment]
This issue was closed due to no response.

**Comment by hyuri** [comment]
Yes, please add this. Ctrl + Y for Redo is absolutely irritating. Only Microsoft uses Ctrl + Y for Redo, basically everyone else use Ctrl + Shift + Z.

We can have both Ctrl + Y and Ctrl + Shift + Z for Redo, so that people used to Ctrl + Y can keep using it.

**Comment by rsnegirev** [comment]
Hi @hyuri 

This is a feature request - bug number 66575.

Thank you for the report!

And we'll text here when it will be released.

It's also connected to the Document Server, so we're moving this issue.
